coretex/cli/modules/node.py

```diff
@@ -236,6 +236,32 @@ def selectNodeMode(storagePath: str) -> Tuple[int, Optional[int]]:
     return availableNodeModes[selectedMode], None
 
 
+def promptCpu(config: Dict[str, Any], cpuLimit: int) -> int:
+    cpuCount: int = clickPrompt("Enter the number of CPUs the container will use (press enter to use default)", config_defaults.DEFAULT_CPU_COUNT, type = int)
+
+    if cpuCount > cpuLimit:
+        errorEcho(f"ERRROR: CPU limit in Docker Desktop ({cpuLimit}) is lower than the specified value ({cpuCount})")
+        return promptCpu(config, cpuLimit)
+
+    return cpuCount
+
+def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
+    nodeRam: int = clickPrompt("Node RAM memory limit in GB (press enter to use default)", config_defaults.DEFAULT_RAM_MEMORY, type = int)
```

Comment:
Default value should be minimum of `config_defaults.DEFAULT_RAM_MEMORY` and `ramLimit`.